MapboxCoreNavigation/RouteController.swift

```diff
@@ -185,7 +185,7 @@ open class RouteController: NSObject {
     /**
      Details about the user’s progress along the current route, leg, and step.
      */
-    public var routeProgress: RouteProgress {
+    @objc public var routeProgress: RouteProgress {
```

Comment:
Besides exposing the property to Objective-C, this change also switches the property’s getter and setter to use Objective-C method dispatch. The `willSet` and `didSet` should continue to work just like before, but it’s worth double-checking to be sure.

Reply:
https://github.com/mapbox/mapbox-navigation-ios/pull/1323/commits/4f21e5a63878a7029c4a125e6890c3ce9c009fa9 verifies that the notification in the property’s `didSet` gets fired.